frontend/src/core/resource/components/ResourceMenu.js

```diff
@@ -0,0 +1,172 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from 'fluent-react';
+import onClickOutside from 'react-onclickoutside';
+
+import './ResourceMenu.css';
+
+import { ResourceItem, ResourcePercent } from '..';
+
+import type { NavigationParams } from 'core/navigation';
+import type { ResourcesState } from '..';
+
+
+type Props = {|
+    parameters: NavigationParams,
+    resources: ResourcesState,
+    navigateToPath: (string) => void,
+|};
+
+type State = {|
+    search: string,
+    visible: boolean,
+|};
+
+
+/**
+ * Render a resource menu for the main navigation bar.
+ *
+ * Allows to switch between resources without reloading the Translate app.
+ */
+export class ResourceMenuBase extends React.Component<Props, State> {
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            search: '',
+            visible: false,
+        };
+    }
+
+    toggleVisibility = () => {
+        this.setState((state) => {
+            return { visible: !state.visible };
+        });
+    }
+
+    // This method is called by the Higher-Order Component `onClickOutside`
+    // when a user clicks outside the user menu.
+    handleClickOutside = () => {
+        this.setState({
+            visible: false,
+        });
+    }
+
+    navigateToPath = (event: SyntheticMouseEvent<>) => {
+        event.preventDefault();
+
+        // $FLOW_IGNORE
+        const path = event.currentTarget.pathname;
+        this.props.navigateToPath(path);
+
+        this.setState({
+            visible: false,
+        });
+    }
+
+    updateResourceList = (event: SyntheticInputEvent<HTMLInputElement>) => {
+        this.setState({
+            search: event.currentTarget.value,
+        });
+    }
+
+    render() {
+        const { parameters, resources } = this.props;
+
+        if (parameters.project === 'all-projects') {
+            return null;
+        }
+
+        let className = 'resource-menu';
+        if (!this.state.visible) {
+            className += ' closed';
+        }
+
+        let resourceName = parameters.resource.split('/').slice(-1)[0];
```

Comment:
I'm not sure I understand this. If there happens to be a resource `path/to/all-resources` then it will match the next condition, but I don't think it should. Why not just check if `parameters.resource` is equal to `all-resources`?